packages/view/backend/pom.xml

```diff
@@ -35,6 +35,7 @@
 		<module>sirius-components-view</module>
 		<module>sirius-components-view-edit</module>
 		<module>sirius-components-view-emf</module>
+		<module>sirius-components-view-emf-widget-reference</module>
```

Comment:
I think it's missing:
`sirius-components-collaborative-formdescriptioneditors-widget-reference`
`sirius-components-collaborative-widget-reference`

Reply:
- sirius-components-collaborative-formdescriptioneditors-widget-reference is referenced in the pom.xml of packages/formdescriptioneditors/backend/pom.xml
- sirius-components-collaborative-widget-reference was already referenced in packages/forms/backend/pom.xml

The pom.xml of each `packages/XXX/backend/pom.xml` are just referencing their content and both projects are in other "packages"